### PR TITLE
Add support for GHA test-commands

### PR DESCRIPTION
**Pull request by icemac**
Plus some other fixed needed for https://github.com/zopefoundation/zope.sqlalchemy/pull/62 resp. fixed while working on that PR.

### CODE REVIEW

File: config/README.rst [hunk 1]
```diff
@@ -268,6 +268,9 @@ updated. Example:
     additional-install = [
         "sudo apt-get update && sudo apt-get install -y libxml2-dev libxslt-dev",
         ]
+    test-commands = [
+        "tox -f ${{ matrix.config[1] }}",
```
Comment (dataflake) [inline config/README.rst:272]: typo: `-e` instead of `-f`

File: config/default/setup.cfg.j2 [hunk 2]
```diff
@@ -44,9 +44,9 @@ ignore-bad-ideas =
 force_single_line = True
 combine_as_imports = True
 sections = FUTURE,STDLIB,THIRDPARTY,ZOPE,FIRSTPARTY,LOCALFOLDER
-known_third_party = %(isort_known_third_party)s
```
Comment (dataflake) [inline config/default/setup.cfg.j2:47]: Was this intended? No spacing after `=` doesn't look very nice.

**Review by dataflake — CHANGES_REQUESTED** [review]

File: config/README.rst (hunk 1 above)
Comment (icemac) [reply config/README.rst:272]: `-f` is intended here, it is an option of https://pypi.org/project/tox-factor/, maybe I should add as an example in the option above.

File: config/default/setup.cfg.j2 (hunk 2 above)
Comment (icemac) [reply config/default/setup.cfg.j2:47]: The space is added in https://github.com/zopefoundation/meta/blob/947b9ba4fbbbae527f79f9c38a4b8c3d86a01b35/config/config-package.py#L177-L184 if needed. Without this change the lines in `setup.cfg` would have trailing whitespace if the options are empty.

**Review by dataflake — APPROVED** [review]
Never heard of tox-factor before. Looks like a nice tool.

**Pull request merged**